### PR TITLE
Parse x-total header get the count of records

### PR DESCRIPTION
**Pull request by vgsantoniazzi**

### CODE REVIEW

**Comment by vgsantoniazzi** [comment]
@NARKOZ Can you take a look? :smile:

**Comment by NARKOZ** [comment]
Sorry, I'll check on weekends.

**Comment by vgsantoniazzi** [comment]
@NARKOZ did you have time to go through this?

**Review by NARKOZ — APPROVED** [review]

**Comment by NARKOZ** [comment]
Thank you ❤️

File: spec/gitlab/headers/total_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+require 'spec_helper'
+
+RSpec.describe Gitlab::Headers::Total do
+  before do
+    @total = described_class.new('x-total' => ' 8 ')
+  end
+
+  describe '.extract_total' do
+    it 'extracts total header appropriately' do
+      expect(@total.total).to eql('8')
```
Comment (jirutka) [inline spec/gitlab/headers/total_spec.rb:12]: Why is `#total` a string and not an integer?

**Pull request merged**